crates/common/tedge_config/src/tedge_config_cli/tedge_config_repository.rs

```diff
@@ -107,4 +81,21 @@ impl<Sources: ConfigSources> TEdgeConfigRepository<Sources> {
             config_defaults: self.config_defaults.clone(),
         })
     }
+
+    // TODO: Explicitly set the file permissions in this function and file ownership!
+    fn store(&self, config: &TEdgeConfig) -> Result<(), TEdgeConfigError> {
+        let toml = toml::to_string_pretty(&config.data)?;
+
+        // Create `$HOME/.tedge` or `/etc/tedge` directory in case it does not exist yet
```

Comment:
The idea to use `$HOME/.tedge` has been fully deprecated. Now, the default tedge config directory is `/etc/tedge` and can be provided on the command line using the `--tedge-config` option.